Modules/posixmodule.c

```diff
@@ -7027,6 +7030,24 @@ parse_file_actions(PyObject *file_actions,
                 }
                 break;
             }
+#ifdef HAVE_POSIX_SPAWN_FILE_ACTIONS_ADDCLOSEFROM_NP
+            case POSIX_SPAWN_CLOSEFROM: {
+                int fd;
+                if (!PyArg_ParseTuple(file_action, "Oi"
+                        ";A closefrom file_action tuple must have 2 elements",
+                        &tag_obj, &fd))
+                {
+                    goto fail;
+                }
+                errno = posix_spawn_file_actions_addclosefrom_np(file_actionsp,
```

Comment:
I do wonder if we'll find platforms where the function exists but returns `EINVAL`. the alpha and beta period hopefully tells us; it won't be one of our tier 1-3 platforms. :)

it is odd that this function returns a posix errno value and that the return value is being assigned into the `errno` thread local global lvalue.  The Solaris family documents the API as doing that, Linux lacks documentation for the API but _appears_ to do the same.